libp2p/wire.nim

```diff
@@ -146,7 +148,8 @@ proc createStreamServer*[T](
       initTAddress(ma).tryGet(), flags, udata, sock, backlog, bufferSize, child, init
     )
   except CatchableError as exc:
-    raise newException(LPError, exc.msg)
+    raise
+      newException(LPError, "Exception in createStreamServer simpler: " & exc.msg, exc)
```

Comment:
Adding `Exception in` seems redundant. The type already conveys this is an exception.
(This comment applies to other messages that contain `Exception in `